### PR TITLE
 Adding doc template to function largestConnCompSliceWise

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Input and output are still to be determined

### CODE REVIEW

File: src/Functions/largestConnCompSliceWise.m [hunk 1]
```diff
@@ -1,6 +1,20 @@
-% Andreas Husch
-% 2018
 function binaryMask = largestConnCompSliceWise(img3d)
+%
+% USAGE:
+%
+%    binaryMask = largestConnCompSliceWise(img3d)
+%
+% INPUT: 
+%    img3d:
```
Comment (Daniel-Tojal) [inline src/Functions/largestConnCompSliceWise.m]: @adhusch Could you please provide a description for the input and output?

**Review by adhusch — CHANGES_REQUESTED** [review]
input: an image volume 
output: binary mask of the largest connected component within that volume, computed slice wise 2d and stacked

**Comment by Daniel-Tojal** [comment]
Thank you, Andreas.

File: src/Functions/largestConnCompSliceWise.m [hunk 2]
```diff
@@ -1,6 +1,21 @@
-% Andreas Husch
-% 2018
 function binaryMask = largestConnCompSliceWise(img3d)
+%
```
Comment (laurentheirendt) [inline src/Functions/largestConnCompSliceWise.m:4]: There is a description missing here.

Comment (Daniel-Tojal) [reply src/Functions/largestConnCompSliceWise.m:4]: Right! There was no description provided in the original file. Could you give us a description of what this function does, @adhusch ?

File: src/Functions/largestConnCompSliceWise.m (hunk 1 above)
Comment (adhusch) [reply src/Functions/largestConnCompSliceWise.m]: Please see my comment from 2 days ago

File: src/Functions/largestConnCompSliceWise.m (hunk 2 above)
Comment (adhusch) [reply src/Functions/largestConnCompSliceWise.m:4]: Please see my review comment from 2 days ago

Comment (Daniel-Tojal) [reply src/Functions/largestConnCompSliceWise.m:4]: I will include this in the description, then, thank you.

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal 👍

**Pull request merged**